src/js/common/schemaform/fields/ArrayField.jsx

```diff
@@ -192,7 +192,9 @@ export default class ArrayField extends React.Component {
     const DescriptionField = typeof description === 'function'
       ? uiSchema['ui:description']
       : null;
-    const isReviewMode = uiSchema['ui:options'].reviewMode;
+
+    const editingState = this.props.uiSchema['ui:options'].reviewMode;
+    const isReviewMode = formData && formData.length && editingState;
```

Comment:
@jbalboni Arrrayfield already had the necessary access to uiSchema, so have simply included editingState here.